### PR TITLE
Relax gstreamer version requirements.

### PR DESCRIPTION
**Pull request by freemangordon**
Use playbin2 "notify::source" signal instead of "signal::source-setup",
which is not available in gstreamer versions prior to 0.10.33.

Signed-off-by: Ivaylo Dimitrov freemangordon@abv.bg

### CODE REVIEW

File: content/media/gstreamer/GStreamerReader.cpp [hunk 1]
```diff
@@ -88,6 +88,8 @@
 
   if (mPlayBin) {
     gst_app_src_end_of_stream(mSource);
+    if(mSource)
```
Comment (romaxa) [inline content/media/gstreamer/GStreamerReader.cpp]: if (
 ^ space here

File: configure.in [hunk 2]
```diff
@@ -5759,7 +5759,7 @@ if test "$MOZ_GSTREAMER"; then
     # core/base release number
     # depend on >= 0.10.33 as that's when the playbin2 source-setup signal was
     # introduced
-    GST_VERSION=0.10.33
```
Comment (romaxa) [inline configure.in:5762]: File bug in bugzilla mozilla org with this patch proposal, after that we can push it here

File: content/media/gstreamer/GStreamerReader.cpp (hunk 1 above)
Comment (freemangordon) [inline content/media/gstreamer/GStreamerReader.cpp]: ok, will fix and update pull request

Comment (freemangordon) [inline content/media/gstreamer/GStreamerReader.cpp]: please, close that pull request, I'll create a new one with relevant commits only. Sorry for the noise :)

**Pull request closed without merging**